recipes-kernel/linux/linux-yocto-onl-linux-6.1.y/bisdn-kmeta/cfg/0001-tun-allow-setting-hwaddr-on-tap-interface-creation.patch

```diff
@@ -0,0 +1,43 @@
+Upstream-Status: undecided if suitable [rather hackish, but also simple]
```

Comment:
This file is missing the change "linux-yocto-onl: move Upstream-Status into commit message" from the 5.15 version.

Reply:
Fixed.